### PR TITLE
Example of implementing Google Analytics 4 with custom destination on web

### PR DESCRIPTION
**Pull request by logason**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…art-custom-destination**](https://docs-3gswq12lf.avo-dev.app/implementation/start-custom-destination) |
|:-:|
|<a href="https://docs-3gswq12lf.avo-dev.app/implementation/start-custom-destination">Screenshot of /implementation/start-custom-destination</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/el9JY_k9dMnoV5IOfG1Nb4x1gXuhrsoZ6Ocr_l0dGdXBhGZML9-4uNDHQryT8_v-ff1OfWwBVqmVRm7hvdoheA.png">(view full size)</a> |







Commit 1f15ee77d4cf6fb82544645f4fb850138b1775e8 (https://docs-3gswq12lf.avo-dev.app).

File: pages/implementation/start-custom-destination.mdx [hunk 1]
```diff
@@ -134,6 +134,45 @@ Call the initAvo function:
...
+
+Here is a more specific example of how you can use a custom destination in Javascript on Web to integrate with your Google Analytics 4 SDK to implement custom event tracking with Avo.
+
+```javascript
+import Avo from './Avo';
+
+// An object representing your Google Analytics destination that you pass into initAvo():
+var googleAnalyticsDestination = {
+  make: function (env) {
+    // If your gtag SDK is not already initialized you can intialize it here
```
Comment (hugihlynsson) [inline pages/implementation/start-custom-destination.mdx:147]: An improvement here could be to add the code needed but commented out.

File: pages/implementation/start-custom-destination.mdx [hunk 2]
```diff
@@ -134,6 +134,45 @@ Call the initAvo function:
 Avo.initAvo({ env: 'dev' }, {}, {}, customDestination);
 ```
 
+#### Javascript to Google Analytics 4
+
+Here is a more specific example of how you can use a custom destination in Javascript on Web to integrate with your Google Analytics 4 SDK to implement custom event tracking with Avo.
+
+```javascript
+import Avo from './Avo';
```
Comment (hugihlynsson) [inline pages/implementation/start-custom-destination.mdx:142]: Could we include insturctions what this looks like if I'm not using a bundler and just including this on my static page, like we're doing on avo.app?

**Review by hugihlynsson — APPROVED** [review]
Nice! Some comments that don't need to be acted on now

File: pages/implementation/start-custom-destination.mdx (hunk 2 above)
Comment (logason) [reply pages/implementation/start-custom-destination.mdx:142]: Yes, let's create a separate section on it in our updated docs ✨

**Pull request merged**